backend/jest.config.js

```
@@ -0,0 +1,33 @@
+/** @type {import('jest').Config} */
+module.exports = {
+  testEnvironment: 'node',
+  // Root is backend/; will discover **/*.test.js by default
+  clearMocks: true,
+  coverageProvider: 'v8',
+  collectCoverage: true,
```

Biome (2.4.4)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)